### ISSUE TITLE
Map Movement Could Be Considered Tedious?

### ISSUE THREAD

**Issue by valkyrienyanko** [Valks-Games/sankari#23]
Right now with map movement you have to tap the right arrow twice to get to the next level. Tapping the right arrow once to get to the next level would be nice. Keep in mind that the map level placements are not just left to right, they could be placed in all 4 directions up, down, left, right. Look at how Super Mario World (SNES) does it. I want to do something similar to that where you tap right arrow (or whatever direction) once, and you see the animation of the player lerping to the new level but always following the path. Or should the game map be kept as is?

**Comment by Smokedelephant** [comment]
Yes we could have a fast travel menu for back tracking and maybe even a acceleration system to make the player move faster in the areas that are long and slow
